src/uct/ib/rc/accel/rc_mlx5.h

```diff
@@ -28,10 +28,15 @@
 enum {
     /* Tag Matching address. It additionally contains QP number which
      * is used for hardware offloads. */
-    UCT_RC_MLX5_IFACE_ADDR_FLAG_TM         = UCS_BIT(0),
+    UCT_RC_MLX5_IFACE_ADDR_FLAG_TM = UCS_BIT(0)
```

Comment:
please also revert it as it was before - using type instead of flag - `UCT_RC_MLX5_IFACE_ADDR_TYPE_TM`
otherwise it is confusing like this field can be a flag

Reply:
hmmm, reverted